### DISCUSSION TITLE
Behaviour of Trivy Analyzer switch "Ignore unfixed vulnerabilities"

### DISCUSSION THREAD

**Discussion by larsriehn** [DependencyTrack/dependency-track#4157] [Q&A]
We use the Trivy analyzer and have imported the SBOM's with "Ignore unfixed vulnerabilities" switched off.
I now set the switch to "Ignore unfixed vulnerabilities", and nothing changed. I expected that during the next daily "Portfolio vulnerability analysis" the number of CVE's would drop and only Trivy CVE's with an existing fix are shown. But this is not the case. All CVE's remain.
Is this the intended behaviour?
Can I achieve a new (clear) CVE evaluation in any way?

**Comment by larsriehn** [comment]
Anyone?

**Comment by fupgang** [comment]
> Can I achieve a new (clear) CVE evaluation in any way?

Workaround: Remove all components, this purges all vulnerabilities. Then upload the sbom again to restore the components and trigger a new analysis.
